pkg/utils/kubernetes/leaderelection_test.go

```diff
@@ -86,7 +86,7 @@ var _ = Describe("LeaderElection", func() {
 	Describe("#ReadLeaderElectionRecord", func() {
 		Context("endpoints lock", func() {
 			BeforeEach(func() {
-				lock = resourcelock.EndpointsResourceLock
+				lock = resourcelock.EndpointsLeasesResourceLock
```

Comment:
Instead of using `EndpointsLeasesResourceLock` just don't use any constant (since there is none anymore) and write "endpoints" explicitly here.